bin/create.js

```diff
@@ -57,6 +74,10 @@ var tokenOptions = {
         description: 'pre-compile a local file using the indicated library (for now only `babel` is supported and will read `.babelrc` if present)',
         type: 'string',
     },
+    auth: {
+      description: 'set webtask permisions',
```

Comment:
This is for authorization, right? We should be more descriptive with what people can say here, give an example or two, even if this is just an e-mail address or e-mail address domain.